Main.py

```diff
@@ -0,0 +1,15 @@
+class Task:
```

Comment:
it should not be inside the main file. you should create a separate python file as we discussed in the meeting this is wrong.